webhook/resources/backup/mutator.go

```diff
@@ -50,6 +53,35 @@ func (b *backupMutator) Create(request *admission.Request, newObj runtime.Object
 	}
 	patchOps = append(patchOps, patchOp)
 
+	backupLabels := make(map[string]string)
+	volumeName, isExist := backup.Labels[types.LonghornLabelBackupVolume]
+	if !isExist {
+		err := errors.Wrap(err, "can not find the backup volume label")
+		return nil, werror.NewInvalidError(err.Error(), "")
+	}
+	volume, err := b.ds.GetVolumeRO(volumeName)
+	if err != nil {
+		err := errors.Wrapf(err, "failed to get volume: %v", volumeName)
+		return nil, werror.NewInvalidError(err.Error(), "")
+	}
+	volumeAccessMode := longhorn.AccessModeReadWriteOnce
+	if volume.Spec.AccessMode != "" {
+		volumeAccessMode = volume.Spec.AccessMode
+	}
```

Comment:
This is somewhat confusing.
```suggestion
	volumeAccessMode := volume.Spec.AccessMode
	if volumeAccessMode == "" {
		volumeAccessMode = longhorn.AccessModeReadWriteOnce
	}
```

Reply:
Thanks, I will follow the suggestion to correct it.